### ISSUE TITLE
deis git:remote just prints "Exit status 129"

### ISSUE THREAD

**Issue by bkoski** [deis/workflow-cli#196]
Not sure what's wrong.  I'm able to manually add the git remote.  But if I attempt to use the built-in deis command to add, I get an error 129 without much explanation:

```
deis git:remote -a MY_APP_NAME -f
Error: exit status 129
```

**Comment by Joshua-Anderson** [comment]
@bk What operating system are you using? I've seen this on debian jessie but that's it. 

The error is coming from git returning a `129`.
I plan to make a PR to fix Sterr printing, so we at least get the error message soon.

**Comment by bkoski** [comment]
I'm on OS X 10.10.5, `git version 2.5.4 (Apple Git-61)`, deis cli `v2.4.0`.

**Comment by Joshua-Anderson** [comment]
Interesting, I don't have this problem with OSX `10.11.6` and `git version 2.7.4 (Apple Git-66)`.

I'll make a PR to log the git Stderr in the next few hours so that we can start debugging.

**Comment by Joshua-Anderson** [comment]
I created PR #199 to add more git output in case of errors.

The PR binary for OS X is https://storage.googleapis.com/workflow-cli-pr/07637bd/deis-07637bd-darwin-amd64

If you're have the time and would like to try and investigate this, run the same command you did before with this binary and hopefully enough information should be logged that we can work from there.

**Comment by bkoski** [comment]
Strangely, even this build returns 129 for me:

```
./deis-07637bd-darwin-amd64 git:remote -a MY_APP_NAME
Error: exit status 129
```

As far as I can tell, all of the other CLI commands are working just fine.

**Comment by Joshua-Anderson** [comment]
Thanks for trying it out! I just added the better error output to `git remote add`, but I realized I should add the error output to every git command. If you have a chance, this
https://storage.googleapis.com/workflow-cli-pr/83abaa4/deis-83abaa4-darwin-amd64
should print the Stderr from git. 

Also your specific error leads me to think that `git remote get-url <remote name>` is where the error is coming from.
